lib/menu_printer.rb

```diff
@@ -1,35 +1,38 @@
+require_relative "font_color_and_style"
+
 class MenuPrinter
-  attr_accessor :stdout
+  attr_accessor :stdout, :colors
 
   def initialize(stdout)
     @stdout = stdout
+    @colors = FontColorAndStyle.new
   end
   
   def project_menu
-    stdout.puts("\e[38;5;40mWelcome to Taskitome!")
-    stdout.puts("\e[0;37m=============================\n")
-    stdout.puts("\e[0mPROJECTS MENU")
-    stdout.puts("\e[0;37m-----------------------------")
-    stdout.puts("\e[40;38;5;214mENTER A COMMAND:\e[0m")
-    stdout.puts("\e[1;37ma   \e[0;35mAdd a new project")
-    stdout.puts("\e[1;37mls  \e[0;35mList all project")
-    stdout.puts("\e[1;37md   \e[0;35mDelete a project")
-    stdout.puts("\e[1;37me   \e[0;35mEdit a project")
-    stdout.puts("\e[1;37mq   \e[0;35mQuit the app\e[0m\n\n")
+    stdout.puts colors.success("Welcome to Taskitome!")
+    stdout.puts colors.default("=============================")
+    stdout.puts ("PROJECTS MENU")
+    stdout.puts colors.default("-----------------------------")
+    stdout.puts colors.alert("ENTER A COMMAND:")
+    stdout.puts colors.default("a ") + colors.options("  Add a new project")
```

Comment:
These series of methods where you have the command and then a description of the command feels like a separate formatting method. `format.menu_item('a', 'Add a new project')`. You can see that there is formatting in the string space characters that could be broken without encapsulating it in the formatter class too.

Reply:
That is a good idea! The string spacing is a little awkward though, all of them have two spaces except for `List all tasks` and `List all projects` have only one space. 🤔